src/eyes.js

```diff
@@ -103,7 +106,15 @@ class Eyes {
 
   async _processPage(t) {
     if (!this._processPageClientFunction) {
-      this._processPageClientFunction = await this._clientFunctionWrapper(processPageAndSerialize);
+      this._processPageClientFunction = await this._clientFunctionWrapper(processPageAndSerialize, {
+        functionArgs: [
+          undefined, // will default processPage to document
+          {
+            dontFetchResources: !!process.env.APPLITOOLS_DISABLE_BROWSER_FETCHING,
```

Comment:
Ideally this should be passed by the user in `eyes.checkWindow` arguments. I will continue the discussion with product in Trello

Reply:
Seems like we'd want an either/or option (global or granular). Let me know where y'all land.